internal/server/v2/event/golden/event_collection/FireEvent_CA_202210.json

```diff
@@ -0,0 +1,295 @@
+{
+  "eventCollection": {
```

Comment:
I feel the response can be simplified. For example eventCollection seems redundant?

propVals could be an array of 
{
 prop: xxx
 vals: []
}

Reply:
The motivation to have a wrapper like this that we can have a "bulk" method in the future.

Currently in V1, EventCollection is not a "bulk" method, and we were planning to add a "bulk" method, that's why we have this wrapper, so that in the future, "eventCollection" can be a repeated field.